### DISCUSSION TITLE
RaspBee II cerco aiuto per installarlo

### DISCUSSION THREAD

**Discussion by Silvertav** [indomus/forum#3095] [HUB: Home Assistant]
Salve a tutti, ho acquistato un RaspBee II, ma nonostante ci abbia passato un sacco di tempo cercando di installarlo, leggendo guide e guardando tutorial non riesco proprio a farlo funzionare.
C'è qualche anima pia che mi può dare una mano o una consulenza? 
-edit-
Grazie

**Comment by KC-inDomus** [comment]
con quale birdge software stai provando a usarla?
